.github/workflows/receive-fork-review.yml

```diff
@@ -17,12 +18,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
-      # This is intended to pass commit ID to run-e2e-on-fork workflow.
+      # This is intended to pass commit id, base_ref, and pr number to run-e2e-on-fork workflow.
       # Reference: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
-      - name: Save commit ID
+      - name: Save commit ID, base ref (ie. target branch), and pull request number
         run: |
           mkdir -p ./commit
+          echo ${{ github.event.pull_request.base.sha }} > ./commit/base_sha
           echo ${{ github.event.review.commit_id }} > ./commit/commit_id
+          echo ${{ github.event.pull_request.number }} > ./commit/pr_number
```

Comment:
We're passing two more data to runner workflow, similar to how we did e2e testing. 

- Base sha is required for dependency review so that it has a sha to compare to.
- PR number is required by codeql to get the ref to post status events too.

Note that these inputs are sanitized at the runner workflow.